lib/sisimai/bite/email/gsuite.rb

```diff
@@ -0,0 +1,273 @@
+              # Diagnostic-Code: smtp; 550 #5.1.0 Address rejected.
+              # Last-Attempt-Date: Fri, 24 Mar 2017 23:34:10 -0700 (PDT)
+              v = dscontents[-1]
+
+              if cv = e.match(/\A[Ff]inal-[Rr]ecipient:[ ]*(?:RFC|rfc)822;[ ]*(.+)\z/)
+                # Final-Recipient: rfc822; kijitora@example.de
+                if v['recipient']
+                  # There are multiple recipient addresses in the message body.
+                  dscontents << Sisimai::Bite.DELIVERYSTATUS
+                  v = dscontents[-1]
+                end
+                v['recipient'] = cv[1]
+                recipients += 1
+
+              elsif cv = e.match(/\A[Aa]ction:[ ]*(.+)\z/)
+                # Action: failed
+                v['action'] = cv[1].downcase
+
+              elsif cv = e.match(/\A[Ss]tatus:[ ]*(\d[.]\d+[.]\d+)/)
+                # Status: 5.0.0
+                v['status'] = cv[1]
+
+              elsif cv = e.match(/\A[Rr]emote-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                # Remote-MTA: dns; 192.0.2.222 (192.0.2.222, the server for the domain.)
+                v['rhost'] = cv[1].downcase
+                v['rhost'] = '' if v['rhost'] =~ /\A\s+\z/  # Remote-MTA: DNS;
+
+              elsif cv = e.match(/\A[Ll]ast-[Aa]ttempt-[Dd]ate:[ ]*(.+)\z/)
+                # Last-Attempt-Date: Fri, 24 Mar 2017 23:34:10 -0700 (PDT)
+                v['date'] = cv[1]
+
+              else
+                if cv = e.match(/\A[Dd]iagnostic-[Cc]ode:[ ]*(.+?);[ ]*(.+)\z/)
+                  # Diagnostic-Code: smtp; 550 #5.1.0 Address rejected.
+                  v['spec'] = cv[1].upcase
+                  v['diagnosis'] = cv[2]
+                else
+                  # Append error messages continued from the previous line
+                  if endoferror && ( v['diagnosis'] && v['diagnosis'].size > 0 )
+                    endoferror = true if e.empty?
+                    endoferror = true if e =~ /\A--/
+
+                    next if endoferror
+                    next unless e =~ /\A[ ]/
+                    v['diagnosis'] += e
+                  end
+                end
+              end
+
+            else
+              # Reporting-MTA: dns; googlemail.com
+              # Received-From-MTA: dns; sironeko@example.jp
+              # Arrival-Date: Fri, 24 Mar 2017 23:34:07 -0700 (PDT)
+              # X-Original-Message-ID: <06C1ED5C-7E02-4036-AEE1-AA448067FB2C@example.jp>
+              if cv = e.match(/\A[Rr]eporting-MTA:[ ]*(?:DNS|dns);[ ]*(.+)\z/)
+                # Reporting-MTA: dns; mx.example.jp
+                next if connheader['lhost'].size > 0
+                connheader['lhost'] = cv[1].downcase
+                connvalues += 1
+
+              elsif cv = e.match(/\A[Aa]rrival-[Dd]ate:[ ]*(.+)\z/)
+                # Arrival-Date: Wed, 29 Apr 2009 16:03:18 +0900
+                next if connheader['date'].size > 0
+                connheader['date'] = 1
```

Comment:
__[rubocop]__
* Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.

<!-- Comment by [SideCI](https://sideci.com) -->